meta-oe/recipes-devtools/guider/guider_3.9.7.bb

```diff
@@ -6,17 +6,17 @@ AUTHOR = "Peace Lee <ipeace5@gmail.com>"
 LICENSE = "GPLv2+"
 LIC_FILES_CHKSUM = "file://LICENSE;md5=2c1c00f9d3ed9e24fa69b932b7e7aff2"
 
-PV = "3.9.6+git${SRCPV}"
+PV = "3.9.7+git${SRCPV}"
 PR = "r0"
 
 SRC_URI = "git://github.com/iipeace/${BPN}"
 #SRCREV = "${AUTOREV}"
-SRCREV = "fef25c41efb9bde0614ea477d0b90bd9565ae0b4"
+SRCREV = "bcf107b73dcd3613cfd8f3f68402547ba8f15701"
 
 S = "${WORKDIR}/git"
 R = "${RECIPE_SYSROOT}"
 
-inherit ${@bb.utils.contains("BBFILE_COLLECTIONS", "meta-python2", "distutils", "", d)}
+inherit distutils
```

Comment:
we cant depend on meta-python2, so if it still needs distutils, then please leave this check in. We want to blacklist it unless meta-python2 is included.

Reply:
yes, I need meta-python2 yet...